internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -206,26 +207,63 @@ func (r *BundleReconciler) ensureUnpackPod(ctx context.Context, bundle *rukpakv1
 		pod.Spec.AutomountServiceAccountToken = &automountServiceAccountToken
 		pod.Spec.Volumes = []corev1.Volume{
 			{Name: "util", VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}}},
+			{Name: "manifests", VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}}},
 		}
 		pod.Spec.RestartPolicy = corev1.RestartPolicyNever
-		if len(pod.Spec.InitContainers) != 1 {
-			pod.Spec.InitContainers = make([]corev1.Container, 1)
+		if bundle.Spec.Source.Type == "git" {
+			// Note: use host networking (hack) so we can resolve https://github.com references in the
+			// Bundle's spec.Source.Git.Repository. Maybe we should revisit having the user inline https://github.com?
+			pod.Spec.HostNetwork = true
```

Comment:
I'm surprised this is necessary. What error are you running into? By default, the dnsPolicy for pods is `ClusterFirst`, which seems fine? 

https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy

Reply:
I took it out and the e2e still passed so I assume we are fine without this setting.